pkg/controllers/dashboard/chart/chart.go

```diff
@@ -24,6 +24,9 @@ const (
 
 	// ProvisioningCAPIChartName name of the chart for rancher-provisioning-capi.
 	ProvisioningCAPIChartName = "rancher-provisioning-capi"
+
+	// FleetChartName name of the chart for fleet.
+	FleetChartName = "fleet"
```

Comment:
This is already defined in [pkg/fleet/const.go](https://github.com/rancher/rancher/blob/release/v2.8/pkg/fleet/const.go#L9)

Reply:
Oh I didn't notice, thanks!